.github/workflows/pull-request.yaml

```diff
@@ -63,3 +64,20 @@ jobs:
     # See https://docs.github.com/en/actions/using-workflows/reusing-workflows to learn more.
     # See  https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions
     # for the security recommendations.
+
+  comment-pr:
+    name: Comment PR with Audit results
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download artifact from Build
+      uses: actions/download-artifact@v3 
```

Comment:
Can you please pin this workflow with the hash digest of the latest like everywhere else in the workflows?